deployments/03-instrumented-app.yml

```diff
@@ -34,6 +34,7 @@ spec:
           imagePullPolicy: IfNotPresent
           securityContext:
             runAsUser: 0
+            privileged: true
```

Comment:
This is required only on some hosts, while others work properly with just the capability. I'd maybe just add this commented with a message:
```yaml
    # uncomment the following line if the logs still show permission errors
    # privileged: true
```